include/evmc/evmc.hpp

```diff
@@ -56,8 +56,8 @@ class result : private evmc_result
     /// releases the ownership of the resources and invalidates this object.
     evmc_result raw() noexcept
     {
-        auto out = evmc_result{};
-        std::swap<evmc_result>(out, *this);
+        const auto out = evmc_result{*this};  // Copy data.
```

Comment:
Why is the `const` needed here?

Reply:
It is not needed, just good practice (I believe).